### ISSUE TITLE
PROTOCOL = for tnsnames.ora doesn't have a default value

### ISSUE THREAD

**Issue by bschonec** [biemond/biemond-oradb#295]
It would be great if the tnsnames.ora would default to PROTOCOL=TCP.  Currently, if you don't specify a protocol the tnsnames.ora file looks like:

```
connection1.example.com =
  (DESCRIPTION =
    (ADDRESS = (PROTOCOL = )(HOST = oracle1.example.com)(PORT = 1700))
    (CONNECT_DATA =
      (SERVER = dedicated)
      (SERVICE_NAME = oracle1.example.COM)
    )
  )
```

**Comment by bschonec** [comment]
See merge request #296

**Comment by bschonec** [comment]
Resolved by #296
